site/content/docs/manifest/environment.en.md

```diff
@@ -159,6 +159,25 @@ List of [public AWS Certificate Manager certificate](https://docs.aws.amazon.com
 By attaching public certificates to your load balancer, you can associate your Load Balanced Web Services with a domain name and reach them with HTTPS. 
 See the [Developing/Domains](../developing/domain.en.md#use-domain-in-your-existing-validated-certificates) guide to learn more about how to redeploy services using [`http.alias`](./lb-web-service.en.md#http-alias).
 
+<span class="parent-field">http.public.</span><a id="http-public-access-logs" href="#http-public-access-logs" class="field">`access_logs`</a> <span class="type">Boolean or Map</span>   
+Enable Elastic Load Balancing access logs.   
+If you specify `true`, Copilot will create an S3 bucket to which the Public Load Balancer will store access logs.
+
+```yaml
+http:
+  public:
+    access_logs: true 
+```
+
+You can also specify an existing S3 bucket with a map.
+```yaml
+http:
+ public:
+   access_logs:
+     bucket_name: my-bucket
+     bucket_prefix: access-logs
+```
+
```

Comment:
Similarly we might want to have a section for each of `bucket_name` and `bucket_perfix`!

Reply:
Ooo